src/main/java/com/cburch/logisim/fpga/download/AlteraDownload.java

```diff
@@ -476,7 +481,7 @@ private boolean CreateJicFile() {
     var command = new ArrayList<String>();
     command.add(alteraVendor.getBinaryPath(3));
     command.add("-c");
-    command.add((ScriptPath + AlteraCofFile).replace(ProjectPath, "../"));
+    command.add((scriptPath + AlteraCofFile).replace(ProjectPath, "../"));
     final var Jic = new ProcessBuilder(command);
     Jic.directory(new File(SandboxPath));
     try {
```

Comment:
Maybe Jic should be changed to jic

Reply:
I tried to resist from fixing this type of issues with this PR to not distract the reviewer from the main subject. And while I apparently failed in some places, I think the less irrelevant changes the better. This PR review already requires enough attention to check the converted HDL that will leave this as is now.